ui/00_message/examine_message/common/common_examine.xml

```diff
@@ -50,21 +50,27 @@
   </message>
   <message>
     <original>何かが引っ掛かって開かない</original>
+    <translation>It seems to be caught on something.</translation>
   </message>
   <message>
     <original>厳重に封印されている――</original>
+    <translation>It has been strongly sealed.</translation>
   </message>
   <message>
     <original>文字はかすれていて読めない</original>
+    <translation>The characters are faint and cannot be read.</translation>
   </message>
   <message>
     <original>すでに息絶えている――</original>
+    <translation>Out of breath.</translation>
```

Comment:
Given the `――` it might make more sense to be `Out of breath...` or `Out of breath-`.